main/internal/implementations/resources_writer/terraform_resources_writer.go

```diff
@@ -94,7 +94,7 @@ func (w *TerraformResourceWriter) commitChangesOpenPullRequest(ctx context.Conte
 
 	logrus.Debugf("[commit_changes_open_pull_request] prURL: %v", prURL)
 	w.dragonDrop.PostLog(ctx, "Done opening a pull request for changes made.")
-	return prURL, nil
+	return "prURL", nil
```

Comment:
Should this be in quotes? Could this mess up links we store to PRs on the managed platform?